tests/unit/moduleapi/cluster.tcl

```diff
@@ -197,8 +197,8 @@ start_cluster 2 2 [list config_lines $modules] {
 
         assert_replication_stream $repl {
             {multi}
-            {del count_dels_{4oi}}
             {keyspace.incr_dels}
+            {del count_dels_{4oi}}
```

Comment:
Wonder if this can somehow be a breaking change. The logical order on primary is first the del and then the notification but the replication order is first the notification and then the del. That said, we are not suppose to write inside a key space notification so its probably harmless...

Reply:
see section (4) in top comment

and again, before August 2022 we didn;t emit any KSN in that case